extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1123,6 +1123,17 @@ export class AmpStoryPage extends AMP.BaseElement {
       return this.element.getAttribute('i-amphtml-return-to');
     }
 
+    const navigationPath = this.storeService_.get(
+      StateProperty.NAVIGATION_PATH
+    );
+    const previousPageId = navigationPath[navigationPath.length - 2];
```

Comment:
Since `-2` is not a common paradigm, maybe leave a comment as to where this comes from?

Reply:
Done